core/src/main/resources/hudson/security/GlobalSecurityConfiguration/index.groovy

```diff
@@ -26,6 +26,23 @@ l.layout(norefresh:true, permission:app.ADMINISTER, title:my.displayName, csscla
             set("descriptor", my.descriptor);
 
             f.optionalBlock( field:"useSecurity", title:_("Enable security"), checked:app.useSecurity) {
+                f.entry (title:_("Disable remember me"), field: "disableRememberMe") {
+                    f.checkbox()
+                }
+
+                f.entry(title:_("Access Control")) {
+                    table(style:"width:100%") {
```

Comment:
Bad idea to use inline CSS style but certainly Jenkins project is full of them 😢